_config.yml

```diff
@@ -63,6 +63,12 @@ authors:
     email: duana at OpenTechSchool dot org
     image: http://www.opentechschool.org/images/team/duana.jpg
     about: is Software Developer at Soundcloud and initiator of the Learners-meetup concept. As such she is organising this and other OpenTechSchool events.
+  florian:
+    name: Florian Gilcher
+    gplus: 103019814974030696862
+    email: florian at OpenTechSchool dot org
+    image: http://www.opentechschool.org/images/team/florian.jpg
```

Comment:
Yup, this probably needs a PR to our main [OTS site repository](http://github.com/OpenTechSchool/www.opentechschool.org).